homeassistant/components/airzone/sensor.py

```diff
@@ -0,0 +1,95 @@
+"""Support for the Airzone sensors."""
+from __future__ import annotations
+
+from typing import Final
+
+from aioairzone.const import AZD_HUMIDITY, AZD_NAME, AZD_TEMP, AZD_ZONES
+
+from homeassistant.components.sensor import (
+    STATE_CLASS_MEASUREMENT,
+    SensorEntity,
+    SensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    DEVICE_CLASS_HUMIDITY,
+    DEVICE_CLASS_TEMPERATURE,
+    PERCENTAGE,
+    TEMP_CELSIUS,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import AirzoneEntity
+from .const import DOMAIN
+from .coordinator import AirzoneUpdateCoordinator
+
+SENSOR_TYPES: Final[tuple[SensorEntityDescription, ...]] = (
+    SensorEntityDescription(
+        device_class=DEVICE_CLASS_TEMPERATURE,
+        key=AZD_TEMP,
+        name="Temperature",
+        native_unit_of_measurement=TEMP_CELSIUS,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        device_class=DEVICE_CLASS_HUMIDITY,
+        key=AZD_HUMIDITY,
+        name="Humidity",
+        native_unit_of_measurement=PERCENTAGE,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Add Airzone sensors from a config_entry."""
+    coordinator = hass.data[DOMAIN][entry.entry_id]
+
+    sensors = []
+    for zone_id, zone_data in coordinator.data[AZD_ZONES].items():
```

Comment:
```suggestion
    for system_zone_id, zone_data in coordinator.data[AZD_ZONES].items():
```

For clarity, it would be better to rename this so its clear its the system id and zone id

Reply:
Changed in 18681e4e4cea7e9d8fb226eb3ff9fa963507ad1e